restli-tools/src/test/java/com/linkedin/restli/tools/snapshot/circular/CircularResource.java

```diff
@@ -31,7 +31,7 @@
 @RestLiActions(name="circular")
 public class CircularResource
 {
-  @Action(name="test")
+  @Action(name="test", readOnly = true)
```

Comment:
have you considered post finder?

Reply:
We already talked offline with @mchen07 and @maximelamure as to why finder would not work here.